packages/actor-query-operation-filter-sparqlee/lib/ActorQueryOperationFilterSparqlee.ts

```diff
@@ -5,9 +5,9 @@ import {
   ActorQueryOperationTypedMediated,
 } from '@comunica/bus-query-operation';
 import type { IActorTest } from '@comunica/core';
+import { AsyncEvaluator, isExpressionError } from '@comunica/sparqlee/lib';
```

Comment:
Can you double-check all imports to sparqlee? We should only import from `@comunica/sparqlee` directly, never from `@comunica/sparqlee/lib` or so.